classes/item.rb

```diff
@@ -14,8 +14,6 @@ def move_to_archive
     @archived = true if can_be_archived?
   end
 
-  private
```

Comment:
Hi @fobadara 
Is the method `can_be_archived?` going to be called from outside the Item class?